docs/feature-store/sources-targets.md

```diff
@@ -0,0 +1,76 @@
+| [mlrun.datastore.CSVTarget](https://storey.readthedocs.io/en/latest/api.html#storey.targets.CSVTarget)        | Writes events to a CSV file (offline target).                                                                       | Y      | Y     | Y      |
+| [mlrun.datastore.KafkaTarget](https://storey.readthedocs.io/en/latest/api.html#storey.targets.KafkaTarget)    | Writes all incoming events into a Kafka stream (online target).                                                    | Y | N | N |
+| [mlrun.datastore.NoSqlTarget](https://storey.readthedocs.io/en/latest/api.html#storey.targets.NoSqlTarget)    | The default online target. Persists the data in V3IO table to its associated storage by key (online target).       | Y      | Y     | Y      |
+| mlrun.datastore.RedisNoSqlTarget                                                                              | Persists the data in Redis table to its associated storage by key (online target).                                 | Y      | Y     | N      |
+| mlrun.datastore.SqlTarget                                                                                     | The default offline target. Persists the data in SQL table to its associated storage by key (offline target).      | Y      | N     | Y      |
+| [mlrun.datastore.ParquetTarget](https://storey.readthedocs.io/en/latest/api.html#storey.targets.ParquetTarget)| The Parquet target storage driver, used to materialize feature set/vector data into parquet files (online target). | Y      | Y     | Y      |
+| [mlrun.datastore.StreamTarget](https://storey.readthedocs.io/en/latest/api.html#storey.targets.StreamTarget)  | Writes all incoming events into a V3IO stream (offline target).                                                    | Y      | N     | N      |
+
+## ParquetTarget
+
+### Partitioning
+
+When writing data to a {py:meth}`~mlrun.datastore.ParquetTarget`, you can use partitioning. Partitioning organizes data 
+in Parquet files by dividing large data sets into smaller and more manageable pieces. The data is divided
+into separate files according to specific criteria, for example: date, time, or specific values in a column.
+Partitioning, when configured correctly, improves read performance by reducing the amount of data that needs to be 
+processed for any given function, for example, when reading back a limited time range with `get_offline_features()`.
+
+When using the pandas engine for ingestion, pandas incurs a maximum limit of 1024 partitions on each ingestion.
+If the data being ingested spans over more than 1024 partitions, the ingestion fails.
+Decrease the number of partitions by filtering the time (for example, using start_filter/end_filter of the 
+{py:meth}`~mlrun.datastore.ParquetSource`), and/or increasing the `time_partitioning_granularity`.
+
+storey processes the data row by row (as a streaming engine, it doesn't get all the data up front, so it needs to process row by row). 
```

Comment:
On another PR should fix/add:
* Fix: storey => Storey
* Fix: storey does not have the 1024 limitation =>  storey does not have the 1024 partitions limitation
* Add: Spark does not have the 1024 partitions limitations as well